src/engine/strat_engine/filesystem.rs

```diff
@@ -150,19 +150,38 @@ impl StratFilesystem {
     }
 
     /// Get the mount_point for this filesystem
```

Comment:
If there are multiple possible mountpoints and you are looking for any one of them, then the definite article is a poor choice here. I think the header could stand to be updated a bit.